src/locales/cs.ts

```diff
@@ -204,141 +204,140 @@ export default {
     angela_merici_virgin: 'Sv. Anděly Mericiové, panny',
-    joachim_and_anne_patroness_of_the_province_of_quebec_parents_of_mary: 'Sv. Anny, patronky provincie Québec, Jáchyma, rodičů Panny Marie',
-  //   anno_of_cologne_bishop: 'Saint Anno of Cologne, Bishop',
+    joachim_and_anne_patroness_of_the_province_of_quebec_parents_of_mary: 'Sv. Anny, patronky provincie Québec, a Jáchyma, rodičů Panny Marie',
+    anno_of_cologne_bishop: 'Sv. Anna Kolínského, biskupa',  // TODO: Or `z Kolína`?
     anselm_of_canterbury_bishop: 'Sv. Anselma, biskupa a učitele církve',
     ansgar_of_hamburg_bishop: 'Sv. Ansgara, biskupa',
     anthony_mary_claret_bishop: 'Sv. Antonína Marie Klareta, biskupa',
     anthony_of_egypt_abbot: 'Sv. Antonína, opata',
     anthony_of_padua_priest: 'Sv. Antonína z Padovy, kněze a učitele církve',
-  //   anthony_of_saint_anne_galvao_priest: 'Saint Anthony of Saint Anne Galvão, Priest',
-  //   anthony_of_the_caves_monk: 'Saint Anthony of the Caves, Monk',
+    anthony_of_saint_anne_galvao_priest: 'Sv. Antonína od sv. Anny Galvão, kněze',
+    anthony_of_the_caves_monk: 'Sv. Antonína Kyjevského, mnicha',  // TODO: Or rather `Pečerského`?
     anthony_zaccaria_priest: 'Sv. Antonína Marie Zaccaria, kněze',
     apollinaris_of_ravenna_bishop: 'Sv. Apolináře, biskupa a mučedníka',
-  //   asaph_of_wales_bishop: 'Saint Asaph, Bishop',
-  //   asicus_of_elphin_bishop: 'Saint Asicus, Bishop',
+    asaph_of_wales_bishop: 'Sv. Asafa, biskupa',
+    asicus_of_elphin_bishop: 'Sv. Asika, biskupa',
     athanasius_of_alexandria_bishop: 'Sv. Atanáše, biskupa a učitele církve',
-  //   attracta_of_killaraght_virgin: 'Saint Attracta, Virgin',
+    attracta_of_killaraght_virgin: 'Sv. Atrakty, panny',
     augustine_of_canterbury_bishop: 'Sv. Augustina z Canterbury, biskupa',
     augustine_of_hippo_bishop: 'Sv. Augustina, biskupa a učitele církve',
     augustine_zhao_rong_priest_and_companions_martyrs: 'Sv. Augustina Žao Ronga, kněze, a druhů, mučedníků',
     augustine_zhao_rong_priest: 'Sv. Augustina Žao Ronga, kněze a mučedníka',
-  //   barbara_of_heliopolis_virgin: 'Saint Barbara, Virgin and Martyr',
+    barbara_of_heliopolis_virgin: 'Sv. Barbory, panny a mučednice',
     barnabas_apostle: 'Sv. Barnabáše, apoštola',
     bartholomew_apostle: 'Sv. Bartoloměje, apoštola',
-  //   beatrice_da_silva_meneses_virgin: 'Saint Beatrice da Silva Meneses, Virgin',
+    beatrice_da_silva_meneses_virgin: 'Sv. Beatrice da Silva Meneses, panny',
     bede_the_venerable_priest: 'Sv. Bedy Ctihodného, kněze a učitele církve',
     benedict_of_nursia_abbot: 'Sv. Benedikta, opata',
-  //   benno_of_meissen_bishop: 'Saint Benno of Meissen, Bishop',
-  //   bernadette_soubirous_virgin: 'Saint Bernadette Soubirous, Virgin',
+    benno_of_meissen_bishop: 'Sv. Benna z Míšně, biskupa',
+    bernadette_soubirous_virgin: 'Sv. Bernadety Soubirousové, panny',  // TODO: Or rather `Soubirous`?
```

Comment:
Both are in use, but `Sv. Bernadety Soubirous` (uninflected) seems to be more wide-spread and I would prefer it.